docs/becoming-a-maintainer/getting-practical.md

```diff
@@ -146,6 +146,8 @@ It's essential to have a code of conduct to ensure that everyone feels welcome a
 - [ ] Copy and paste the code of conduct you'd like to use. We recommend using the [Contributor Covenant](https://www.contributor-covenant.org/).
 - [ ] Add a link to the code of conduct in your README and CONTRIBUTING files.
 
+!>  GitHub provides a code of conduct template. To learn more, check out their tutorial, ["Adding a code of conduct using a template"](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project#adding-a-code-of-conduct-using-a-template).
+
 !> Make sure you attribute the code of conduct to the original author.
```

Comment:
Hey @CBID2,

Thanks for the PR! ✨ 

I have thoughts here.

Line 151 is the extended of the checklists. It's there to explain that when folks copy and paste a CoC that like to use from another project, they should attribute it to the original author.

So:
- Having the note in line 149 makes the whole things here out of the flow.
- As this chapter contains tutorials to get practical, a note to check out GitHub CoC template feels like an orphan here. Unless, you can make the wording as part of the tutorial.
- I think we can avoid having a back to back warning admonitions (warning highlights) for better read and understanding. 

I have some ideas on how to implement this, but I need to reread this whole section and another chapter that related to this section. And I won't be able to get back to this until tomorrow.

In the mean time, please let me know what you think about the feedback and if you have ideas on how to implemet the addition.

Thank you! 🙂

Reply:
Hi @adiati98. I followed @BekahHW's method to formatting this information. Check it out and let me know what you think.

Reply:
@CBID2 have you checked out Bekah's suggestion to also add it in the initial COC section as a tip?
Because when I said 'another chapter that relates to this section', I was thinking the same way. 🙂

Reply:
I added it to the initial tip @adiati98